### DISCUSSION TITLE
Toast Position (🍑 statt 🍆)

### DISCUSSION THREAD

**Discussion by drive-n-code** [Traewelling/traewelling#2097] [Ideas / Feature Requests]
Die Position der Notifications / des Toasts am oberen Bildschirmrand ist eher unglücklich gewählt.
Bei jeder Notification werden die wichtigsten Navigationselemente von Traewelling verdeckt.
Wenn man dicke Finger (so wie ich) hat, schafft man es nicht das 'X' zum Schließen des Notification wegzudrücken.
Meistens ist das Resultat nerviges warten bis zum automatischen Ausblenden, um Traewelling dann weiternutzen zu können.

Beispiel:
Nicht möglich Navigationsbar von Traewelling zu nutzen
![image](https://github.com/Traewelling/traewelling/assets/6603422/cbc7b764-b9ed-4030-a237-e195a1df6ac0)

Beispiel:
Nicht möglich die Notification List zu schließen oder die Navigationsbar von Traewelling zu nutzen
![image](https://github.com/Traewelling/traewelling/assets/6603422/bd8b9c9a-7b54-48c7-abb6-75f5220edb39)

Vorschlag: Den Toast vom oberen Bildschirmrand an den unteren Bildschirmrand verschieben.
Dort behält er seine Informationswirkung, verdeckt aber keine wichtigen Navigationselemente. Der Nutzer wird in seiner User Experience nicht gestört.

--

(Betriebssysteme wie Android haben ihre Notifications/Toast übrigens standardmäßig auch im unteren Bildschirmbereich)
![image](https://github.com/Traewelling/traewelling/assets/6603422/7e094023-e6a6-42d6-bbb8-5302e05ce9ae)

**Comment by jeyemwey** [comment]
Sehe ich genauso, hatte aber nie die Energie, mich darüber aufzuregen. [Diese Zeile](https://github.com/Traewelling/traewelling/blob/develop/resources/js/app.js#L19) scheint zu beeinflussen, wo die notifications dran kleben. Ich bin mir aber nicht sicher, ob man das für mobile und Desktop unterschiedlich machen kann.

**Reply by HerrLevin** [reply]
In der Doku stand dazu nichts. Hab den notyf ui-breakpoint von 480px für dieses wundervolle Ding genommen. :D https://github.com/Traewelling/traewelling/pull/2116/files#diff-583a1f586bc298813341bce6fec77a1f6338f2559471fe600f15a18316fe82bdR19

**Comment by MrKrisKrisu** [comment]
@HerrLevin resolved with #2116?
